examples/nats-bench.go

```diff
@@ -102,14 +103,11 @@ func runPublisher(startwg, donewg *sync.WaitGroup, opts nats.Options, numMsgs in
 
 	args := flag.Args()
 	subj := args[0]
-	var msg []byte
-	if msgSize > 0 {
```

Comment:
this doesn't make sense as it's the long version of: `var msg = make([]byte, 0)`